src/lib/components/Navbar.svelte

```diff
@@ -47,9 +42,9 @@
 	</div>
 
 	<div
-		class="fixed inset-0 bg-sucu-pink-01 flex flex-col items-end justify-end space-y-8 p-8 text-white transform duration-300 ease-in-out"
```

Comment:
you might as well have to change this from `fixed` to `absolute` because when it is `fixed` it will be like this

Reply:
https://github.com/user-attachments/assets/bea7ac6e-4afd-4397-b390-7dfd5c7bc1c7

Reply:
https://github.com/user-attachments/assets/496290cc-4296-4029-a5e3-45115ebfe89d

Reply:
@punchanabu Fixed, you can check it again!